### DISCUSSION TITLE
Issue with missing object files during WW3 compilation

### DISCUSSION THREAD

**Discussion by magistrmw228** [NOAA-EMC/WW3#1213] [General]
Hello! 

I'm facing the same problem as the user _NS-Nik_ (https://github.com/NOAA-EMC/WW3/discussions/695) while folowing the WW3 installation manual  and [Quick-start](https://github.com/NOAA-EMC/WW3/wiki/Quick-Start).

As I understand, **this problem is still not solved**. My log file of execution "w3_make" is [make.log](https://github.com/NOAA-EMC/WW3/files/14890484/make.log). 
This is a part from my ".bashrc":

```
# WW_III settings
export WWATCH3_NETCDF=NC4
export NETCDF_CONFIG=/usr/bin/nc-config
export WWATCH3_ENV=~/WW3/model/bin/wwatch3.env

```

 I have all the necessary libraries. 
I can't understand why object files weren't created during the download from _github_ or _model setup_ ("./bin/w3_setup ~/WW3/model -c Gnu -s NCEP_st4"). 

Please help me. I will be happy to provide any additional information (logs, screenshots, etc) if needed.

**Comment by mickaelaccensi** [comment]
Hello,

Please use the cmake procedure to compile WW3. See README here : https://github.com/NOAA-EMC/WW3/blob/develop/model/README.md

The documentation need to be updated.

Feel free to close the issue it works for you.

Best,

**Reply by magistrmw228** [reply]
Thank you. Your advice helped me to go in the right direction and solve the problem faster.
